slave/slave_files/files/home/jenkins-slave/cleanup_docker_images.py

```diff
@@ -73,7 +73,7 @@ def print_progress(args):
         processed_images.add(i)
         print("removing image %s" % i)
         remove_docker_image(i)
-    except subprocess.CalledProcessError, ex:
+    except subprocess.CalledProcessError as ex:
```

Comment:
The exception variable `ex` is not used. Either remove it or even better: output the error message rather then ignoring it.